manifests/cinder-csi-plugin/csi-secret-cinderplugin.yaml

```diff
@@ -1,9 +0,0 @@
-# This YAML file contains secret objects,
-# which are necessary to run csi cinder plugin.
-
```

Comment:
do not delete this file, as CSI can be deployed independently without Cloud controller manager, just change the name of the secret to have consistency. and In CSI doc just add a note to skip secret creation if CCM is deployed already